packages/libs/lambda-at-edge/tests/default-handler/default-handler.test.ts

```diff
@@ -186,6 +199,35 @@ describe("Lambda@Edge", () => {
 
         expect(response.status).toEqual("200");
       });
+      
+      it("handles preview mode", async () => {
+        const event = createCloudFrontEvent({
+          uri: "/preview",
```

Comment:
I think you would need to append trailing slash if `trailingSlash` is true to the URI